### PR TITLE
Add linters

### PR DESCRIPTION
**Pull request by Laylo309**
- Adding JS linters
- Using Javascript
- Creating JS file
- Making hamburger button work
- Adding close button and making it work

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  // Adding listeners to li tags
+  const sections = document.querySelectorAll('.mobile-menu-li');
+  for (let i = 0; i < sections.length; i += 1) {
+    sections[i].addEventListener('click', (event) => {
+      div.remove();
+      UnBlurSections();
+      window.location.hash = `section-${event.target.id}`;
+    });
+  }
+});
```
Comment (arslanbisharat) [inline js/main.js:45]: - [ ] When the user click (or tap) the hamburger button, the mobile menu appears.
- [ ] When the user click (or tap) the close (X) button, the mobile menu disappear.
- [ ] When the user click (or tap) any option of the mobile menu, the mobile menu disappear.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested , 

Hi,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by SpaYco — COMMENTED** [review]
Hello 👋,

This is an additional review as a result of the second opinion request.

Can you explain more about how the CR didn't tell you what to do? I can see the changes that should be done:

> - [ ]  When the user click (or tap) the hamburger button, the mobile menu appears.
> - [ ]  When the user click (or tap) the close (X) button, the mobile menu disappear.
> - [ ]  When the user click (or tap) any option of the mobile menu, the mobile menu disappear.

If you mean to walk you through the steps to fix the code, please note that CRs are not supposed to give you ways to fix things, they are only here to do code reviews, if you're stuck on something and need help, please ask the question on [Discourse](https://questions.microverse.org/), or [schedule a meeting with a Senior CR](https://microverse.zendesk.com/hc/en-us/articles/360026625973-How-can-I-get-help-from-TSEs-#), but I highly recommend you try to see how to fix the issue few times before going there, so you learn debugging skills!

I can see that the requirements are missing the why element though, so I'll make sure the CR knows about it!

Best of luck with future projects, keep rocking 🚀!

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Laylooo @mateo951  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

shine team

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**